EINVALIDTAGNAME when running `npm install react-tinacms-inline`

<!--
  Please fill out each section below, otherwise your issue will be closed. This info allows Tina maintainers to diagnose (and fix!) your issue as quickly as possible.

  Before opening a new issue, please search existing issues: https://github.com/tinacms/tinacms/issues
-->

## Description

EINVALIDTAGNAME when running `npm install react-tinacms-inline`. Based on the error output looks like `react-final-form` is being specified incorrectly within a `package.json` somewhere.

### Steps to reproduce

1. mkdir tmp
2. cd tmp
3. npm init
4. npm install react-tinacms-inline

### Expected result

npm install finishes successfully

### Actual result

What actually happened?

```
~/git/personal/tmp  ➜  npm install react-tinacms-inline
npm ERR! code EINVALIDTAGNAME
npm ERR! Invalid tag name "^>=6": Tags may not have any characters that encodeURIComponent encodes.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/user/.npm/_logs/2021-01-25T17_19_58_889Z-debug.log

```

### Environment

#### Tinacms Package Versions
<!--
Run the following command to get the list of tinacms-package versions in your node_modules:

    npm ls | grep tinacms | grep -v deduped
-->

```text
├── @tinacms/api-git@0.7.0
├── @tinacms/git-client@0.6.7
├── next-tinacms-markdown@0.2.23
└── tinacms@0.21.3
```

<!-- Please add as much information about your environment as possible. -->

#### Tools

| Tool    | Version |
| ------- | ------- |
| browser |         |
| os      |  Darwin computer 20.2.0 Darwin Kernel Version 20.2.0: Wed Dec  2 20:39:59 PST 2020; root:xnu-7195.60.75~1/RELEASE_X86_64 x86_64       |
| node    |   v15.4.0      |
| npm     |   7.0.15      |

--- a/packages/react-tinacms-inline/package.json
+++ b/packages/react-tinacms-inline/package.json
@@ -20,7 +20,7 @@
   "peerDependencies": {
     "react": ">=16.8",
     "react-dropzone": "10.2.1",
-    "react-final-form": "^>=6",
+    "react-final-form": ">=6",
     "react-tinacms-editor": ">=0.0",
     "styled-components": ">=4.1",
     "tinacms": ">=0.13"

